source/jormungandr/tests/journey_common_tests.py

```diff
@@ -943,6 +942,17 @@ def test_shared_section(self):
         assert r['journeys'][0]['sections'][1]['display_informations']['name'] == first_journey_pt
         assert len(r['journeys']) == 1
 
+    if os.getenv('JORMUNGANDR_USE_SERPY'):
+        def test_section_fare_zone(self):
+            """
+            In the 'stop_point' section of a journey, the section 'fare_zone' should be present
+            (only the Serpy serializer has this feature, as Marshall will be deprecated soon)
+            """
+            r = self.query('/v1/coverage/main_routing_test/journeys?to=stopA&from=stopB&datetime=20120614T080100&')
+            assert r['journeys'][0]['type'] == 'best'
+            assert r['journeys'][0]['sections'][1]['type'] == 'public_transport'
+            assert r['journeys'][0]['sections'][1]['from']['stop_point']['fare_zone']
+            assert 'name' in r['journeys'][0]['sections'][1]['from']['stop_point']['fare_zone']
```

Comment:
If it's easily doable, would be nice to add a fare zone and test actual name (also test that no fare_zone is outputed if no data is available)